<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerTractographyDisplayBasicWidget</class>
 <widget class="qSlicerWidget" name="qSlicerTractographyDisplayBasicWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>129</width>
    <height>96</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QCheckBox::indicator {
     width: 21px;
     height: 21px;
 }

 QCheckBox::indicator:checked {
     image: url(:/Icons/Medium/SlicerVisible.png);
 }
QCheckBox::indicator:unchecked {
     image: url(:/Icons/Medium/SlicerInvisible.png);
 }</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QCheckBox" name="LineVisibility">
     <property name="text">
      <string>Line</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="TubeVisibility">
     <property name="text">
      <string>Tube</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="GlyphVisibility">
     <property name="text">
      <string>Glyph</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
